.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,17 @@
+# Note: additional setup is required, see https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
```

Comment:
@NotMyFault it is unlcear what I need to do based on those instructions.   Could perhaps there be a clear section "migrating from RD App to RD Action"?

Reply:
I added this template to the .github repository, it's available as one-click installation on "Actions -> New workflow -> By Jenkins -> Publish changelogs to GitHub Releases".
If you initially setup RD as workflow, you have to create the `release-drafter.yml` file in the .github folder in your repository, according to the guide linked, hence it's available in the default workflow to provide additional guidance if you setup the workflow as described above.

Moving from the App to the workflow is done by merging this PR. The release-drafter.adoc file can be ignored here, no further work is needed from a maintainer's perspective. 
I considered adding a paragraph how to uninstall the App and add the workflow to the guide, however, not every plugin developer has administrative privileges on their repositories, therefore they cannot manage app installations (which does not matter, because the app is not sunset yet) and the guide linked is likely not visited frequently by App installation users to raise awareness of the transition.

Therefore, every repository that has the App installed and does not use CD received a PR like this with the instructions that merging the PR is the simplest way to handle the transition.

If you are interested about the progress, take a look on my gist: https://gist.github.com/NotMyFault/3e3f2da29824b276469f6bb671c7262a